ci-operator/config/openshift/runtime-utils/openshift-runtime-utils-master.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: "7"
+binary_build_commands: make build
```

Comment:
do you still need this since you're not using the `bin` image?

Reply:
I think we should keep it so we ensure `make build` always compiles - unless we want this to live in a separate `test`?

Reply:
you could just do that as part of the unit test command.

Reply:
(or verify)

Reply:
@bparees moved to verify